### PR TITLE
Fixing variant caller

### PR DESCRIPTION
**Pull request by DanielSnellings**
Fixing some compile errors. Major change is that variant calling on giraf files is temporarily disabled as changes to the giraf cigar will require a significant rewrite of the girafCount code.

### CODE REVIEW

File: fastq/pairedEnd.go [hunk 1]
```diff
@@ -46,15 +46,15 @@ func PairEndToChan(readOne string, readTwo string, output chan<- *PairedEnd) {
 	close(output)
 }
 
-func ReadPairBigToChan(readOne string, readTwo string, answer chan<- *PairedEndBig) {
-	var fq *PairedEndBig
+func ReadPairBigToChan(readOne string, readTwo string, answer chan<- PairedEndBig) {
```
Comment (craiglowe) [inline fastq/pairedEnd.go:49]: I think we are both making changes here.  I also needed to change this to get some testing in SimpleGraph to run.  I initially made the same change you did, but after a quick back-and-forth with Eric I added in some code he had, which gave him better performance.  I think we just want to watch this function as we both merge in our changes.

File: giraf/giraf.go [hunk 2]
```diff
@@ -124,7 +124,7 @@ func GirafChanToFile(filename string, input <-chan *Giraf, wg *sync.WaitGroup) {
 }
 
 // GirafPairChanToFile will write a giraf pair end alignment channel to a file
-func GirafPairChanToFile(filename string, input <-chan *GirafPair, wg *sync.WaitGroup) {
+func GirafPairChanToFile(filename string, input <-chan GirafPair, wg *sync.WaitGroup) {
```
Comment (craiglowe) [inline giraf/giraf.go:127]: I think I have the same change.  Something to watch as we both merge.

File: simpleGraph/simpleGraph_test.go [hunk 3]
```diff
@@ -24,10 +24,10 @@ func TestWorkerWithWriting(t *testing.T) {
 	log.Printf("Reading in the genome (simple graph)...\n")
 	log.Printf("Indexing the genome...\n")
 	log.Printf("Making fastq channel...\n")
-	fastqPipe := make(chan *fastq.PairedEndBig, 824)
+	fastqPipe := make(chan fastq.PairedEndBig, 824)
```
Comment (craiglowe) [inline simpleGraph/simpleGraph_test.go:27]: I think I have the same change.  Something to watch as we both merge.

**Review by craiglowe — APPROVED** [review]
Looks great!  I have some identical and similar changes to be able to run "go test" in SimpleGraph

**Review by rimangan — APPROVED** [review]

**Review by rimangan — APPROVED** [review]

**Review by edotau — APPROVED** [review]

**Pull request merged**